app/routes/_index/index.module.scss

```diff
@@ -38,3 +38,15 @@
     font-size: clamp(60px, 6vw, 110px);
     text-transform: uppercase;
 }
+
+.floatingCardBackground {
+    width: 100vw;
+    margin-left: -2%;
+    padding: 7.8% 11.7%;
+    aspect-ratio: 2;
+}
+
+.floatingCardLinkLabel {
+    width: 170px;
+    height: 34px;
```

Comment:
Why fixed dimensions? Why make them update it every time they want to change the text?

Reply:
That's a tricky problem to make `LabelWithArrow` width dynamic

Reply:
What problem?

Reply:
Oh, seems I found the solution 🥳 

[https://jsfiddle.net/6fz4yevk/14/](https://jsfiddle.net/6fz4yevk/14/)

Reply:
https://jsfiddle.net/uc9wvLym/

1. Use aria-hidden.
2. Don't put the hidden element inside of visible.
3. Transition should be defined on the normal state, not on the hover state.